src/main/groovy/com/google/appengine/AppEnginePlugin.groovy

```diff
@@ -245,28 +256,32 @@ class AppEnginePlugin implements Plugin<Project> {
         }
     }
 
-    private void configureAppConfig(Project project, AppEnginePluginExtension appEnginePluginExtension) {
+    private void configureWorkingDirectory(Project project) {
+        AppConfigTaskTemplate.metaClass.getWorkingDirectory = { -> this.explodedAppDirectory }
```

Comment:
The system property vs DSL might be a more difficult question. I think it's a little bit of a preference thing, I suggested the system property so it can be kind of a temporary holdover while we transition everyone to using the output dir by default, without having to maintain both code paths after the transition is complete.